templates/appium.service.j2

```diff
@@ -7,6 +7,7 @@ After=network.target
 Type=simple
 User={{ appium_user }}
 ExecStart=/home/{{ appium_user }}/run-appium
+Environment="PATH=/usr/bin:/usr/share/xcuitrunner:$PATH"
```

Comment:
Variable expansion does not work in `.service` files (see the Stackoverflow post I linked to earlier). You'll need to include the default values for `$PATH`, too.

Reply:
(At the very least, `iproxy`, which is probably in `/usr/bin`, should also be in $PATH)